<?xml version="1.0"?>
<launch>

		<arg name="r_arm_ns"                 default="right_arm"/> 
		<arg name="l_arm_ns"                 default="left_arm"/>

		<arg name="r_robot_base_frame"       default="/calib_right_arm_base_link"/>
		<arg name="l_robot_base_frame"       default="/calib_left_arm_base_link"/>
        
        <arg name="r_model_path"             default="$(find motion_planner)../epfl-task-models/PouringModels"/>
		<arg name="l_model_path"             default="$(find motion_planner)../epfl-task-models/PouringModels"/>


	  <node name="plan2ctrl" pkg="bimanual_motion_planner" type="decoupled_motion_planner" output="screen" respawn="false">
	  	<param name="action_mode" 			 		value="lasa_fixed"/>
	  	<param name="simulation" 			 		value="false" />
	  	<param name="world_frame" 		     		value="$(arg r_robot_base_frame)" />
	  	<param name="model_dt" 				 		value="0.01" />
	  	<param name="reachingThreshold" 	 		value="0.01" />
	  	<param name="orientationThreshold"   		value="0.02" />
		<param name="model_base_path"        		value="$(arg r_model_path)" />
		<param name="topic_ns"        		        value="$(arg r_arm_ns)" />
	  </node>


</launch>
